docs/schemes/local.md

```diff
@@ -65,28 +71,42 @@ Each endpoint is used to make requests using axios. They are basically extending
 To disable each endpoint, simply set it's value to `false`.
 :::
 
-#### `propertyName`
+### `token`
+
+Here you configure the token options.
+
+#### `property`
 
-`propertyName` can be used to specify which field of the response JSON to be used for value. It can be `false` to directly use API response or being more complicated like `auth.user`.
+`property` can be used to specify which field of the response JSON to be used for value. It can be `false` to directly use API response or being more complicated like `auth.token`.
 
-### `tokenRequired`
+#### `required`
 
-This option can be used to disable all token handling. Useful for Cookie only flows. \(Enabled by default\)
+- Default: `true`
 
-### `tokenName`
+This option can be used to disable all token handling. 
+
+::: tip
+Useful for Cookie only flows.
+:::
+
+#### `name`
 
 - Default: `Authorization`
 
-  Authorization header name to be used in axios requests.
+Authorization header name to be used in axios requests.
 
-### `tokenType`
+#### `type`
 
 - Default: `Bearer`
 
- Authorization header type to be used in axios requests.
+Authorization header type to be used in axios requests.
+
+### `user`
```

Comment:
Maybe we can make it `user: { property, autoFetch }`?

Reply:
I totally agree!

Reply:
Refactored in commits https://github.com/nuxt-community/auth-module/pull/548/commits/a56d1eeb7380cb02107ac4757ae6ae436a909340 and https://github.com/nuxt-community/auth-module/pull/548/commits/dfc84b3924ca1c1e73883c3ac870f1787d6a9c5a